hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -40,53 +38,11 @@
 import org.apache.hadoop.hdds.protocol.proto.HddsProtos.NodeState;
 import org.apache.hadoop.hdds.scm.PipelineChoosePolicy;
 import org.apache.hadoop.hdds.scm.PlacementPolicy;
-import org.apache.hadoop.hdds.scm.ScmUtils;
```

Comment:
It was probably the IDE did this, but there is a lot of change in the import ordering. We should revert this, as it will cause merge conflicts.

Reply:
thanks , i have revert these useless change with a force-push